sklearn/svm/classes.py

```diff
@@ -1155,6 +1155,15 @@ class OneClassSVM(BaseLibSVM, OutlierMixin):
         The offset is the opposite of `intercept_` and is provided for
         consistency with other outlier detection algorithms.
 
+    Examples
+    --------
+    >>> from sklearn.svm import OneClassSVM
+    >>> X = [[0], [0.44], [0.45], [0.46], [1]]
```

Comment:
what about something more obvious like what Joel proposed?

Reply:
@NicolasHug  [[-1], [0], [0.5], [1], [10]] ? Predictions for that array are [ 1, -1,  1, -1, -1]

Reply:
So it wasn't really obvious to me how to interpret these predictions, so I tried to find an array which gives more intuitive output.

Reply:
Ok, I was just curious about it. sounds good then, thanks @veerlosar